pkg/operator/ceph/cluster/watcher.go

```diff
@@ -531,10 +534,9 @@ func (c *clientCluster) createNetworkFence(ctx context.Context, pv corev1.Persis
 
 	networkFence := &addonsv1alpha1.NetworkFence{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      fenceResourceName(node.Name, driver),
```

Comment:
`fenceResourceName`  need to be changed to include namespace name where ceph cluster is created or else we might get into problem if we have multiple ceph clusters and as you know NetworkFence is per ceph cluster

Reply:
I see your point to add the namespace to the name in case of multiple cephclusters.

Reply:
Even more reason to consider if we can convert NetworkFence to a namespaced resource...